python/hsfs/core/feature_view_engine.py

```diff
@@ -923,3 +935,138 @@ def _check_if_exists_with_prefix(self, f_name, f_set):
             )
         else:
             return f_name
+
+    def enable_feature_logging(self, fv):
+        self._feature_view_api.enable_feature_logging(fv.name, fv.version)
+        fv.enabled_logging = self.get(fv.name, fv.version).enabled_logging
```

Comment:
Why do we need two requests to the backend? I would assume that if the call to `enable_feature_logging` doesn't raise an error, it went through and then we can just set `fv.enabled_logging` to true.

Reply:
It is to make sure the backend enable logging successfully. Since this method `enable_feature_logging` will only be called once, the impact on the performance is small. But maybe it should check `enabled_logging=True` here.

Reply:
Could it be that the response is 200 but feature logging was not enabled successfully for that feature view? (e.g., if creating one of the feature group fails).

Generally, if enabling feature logging fails in the backend, an exception should be raise and the response of the request would be != 200. So, a 200 response is "verifying" that feature logging was enabled successfully.
If logging_enabled was an object, another approach would be that the backend returns the updated object in the response and hsfs update feature view inside the same method in feature_view_api. But in this case is a boolean, so 200 response would suffice I believe :)

Reply:
ok, set it to True